dockerfile_functions.sh

```diff
@@ -708,10 +708,21 @@ EOI
 
 run_scc_gen() {
     if [[ "${vm}" == "openj9" && "${os_family}" != "windows" ]]; then
-        cat >> "$1" <<-EOI
-        RUN /scripts/generate_openj9_scc.sh
-        ENV OPENJ9_JAVA_OPTIONS="-Xshareclasses:name=openj9_system_scc,cacheDir=/opt/java/.scc,readonly,nonFatal"
-EOI
+        # Alpine needs bash to be present to run the generate_openj9_scc.sh script
+        if [[ "${os_family}" == "alpine" ]]; then
+            cat >> "$1" <<-EOI
+            RUN apk add --no-cache --virtual .scc-deps bash; \\
```

Comment:
`generate_openj9_scc.sh` uses `curl` so adding it to `.scc-deps` along with bash can fix the issue